dev/bin/minikube-start.sh

```diff
@@ -49,7 +49,7 @@ if [ "${disable_extra_components}" != 'true' ]; then
   kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/v0.9.0/manifests/setup/prometheus-operator-0servicemonitorCustomResourceDefinition.yaml
   . "${SCRIPT_DIR_PATH}/knative-installer.sh"
   istioctl manifest apply --set profile=default --set values.gateways.istio-ingressgateway.type="ClusterIP"
-  sed -i -E "s|(.*http://).*(:30007.*)|\1$(minikube -p "${MINIKUBE_PROFILE}" ip)\2|" ${KUSTOMIZE_DIR}/overlays/minikube/istio/jwt-request-authentication.yaml
+  sed -i -E "s|\(.*http://\).*\(:30007.*\)|\1$(minikube -p "${MINIKUBE_PROFILE}" ip)\2|" ${KUSTOMIZE_DIR}/overlays/minikube/istio/jwt-request-authentication.yaml
```

Comment:
```suggestion
  sed -i -E "s|<REPLACE_WITH_MINIKUBE_IP>|$(minikube -p "${MINIKUBE_PROFILE}" ip)|" ${KUSTOMIZE_DIR}/overlays/minikube/istio/jwt-request-authentication.yaml
```

Reply:
It actually works, I must have miswrote something, let me commit it, thanks @kostola

Reply:
I don't get why my suggestion was discarded and then added as a separate commit, but I'm happy it works :+1: